### PR TITLE
feat: create search bar

### PR DESCRIPTION
**Pull request by diego-aquino**
- Created the search bar component.
- Removed unused CSS files.
- Added `@/` as an absolute import.

https://github.com/user-attachments/assets/f88d1ccf-7142-41a8-b897-5b2f4e0fa272

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -97,6 +97,7 @@
     }
   },
   "cli": {
-    "schematicCollections": ["@angular-eslint/schematics"]
+    "schematicCollections": ["@angular-eslint/schematics"],
+    "analytics": "4e54d43f-6f1b-438d-aadf-bdee1605cc24"
```
Comment (pedrohenrique-ql) [inline angular.json:101]: What is this for?

**Review by pedrohenrique-ql — APPROVED** [review]
Nice! I left just one question.

File: angular.json (hunk 1 above)
Comment (diego-aquino) [reply angular.json:101]: Angular asks me if we agree in opting into their analytics. I just said yes to clear the message (Angular asks it on every command) and help them collect telemetry as a fellow library developer. :slightly_smiling_face: 

image

Comment (pedrohenrique-ql) [reply angular.json:101]: Oh, okay :smile:

**Pull request merged**